README.md

```diff
@@ -2,50 +2,40 @@
 [![Crowdin](https://badges.crowdin.net/paperless-ngx/localized.svg)](https://crowdin.com/project/paperless-ngx)
 [![Documentation Status](https://readthedocs.org/projects/paperless-ngx/badge/?version=latest)](https://paperless-ngx.readthedocs.io/en/latest/?badge=latest)
 [![Coverage Status](https://coveralls.io/repos/github/paperless-ngx/paperless-ngx/badge.svg?branch=master)](https://coveralls.io/github/paperless-ngx/paperless-ngx?branch=master)
+[![Chat on Matrix](https://matrix.to/img/matrix-badge.svg)](https://matrix.to/#/#paperless:adnidor.de)
 
-# Paperless-ngx
-
-[Paperless (click me)](https://github.com/the-paperless-project/paperless) is an application by Daniel Quinn and contributors that indexes your scanned documents and allows you to easily search for documents and store metadata alongside your documents.
-
-Paperless-ng is a fork of the original project, adding a new interface and many other changes under the hood. These key points should help you decide whether Paperless-ng is something you would prefer over Paperless:
-
-* Interface: The new front end is the main interface for Paperless-ng, the old interface still exists but most customizations (such as thumbnails for the document list) have been removed.0
-* Encryption: Paperless-ng does not support GnuPG anymore, since storing your data on encrypted file systems (that you optionally mount on demand) achieves about the same result.
-* Resource usage: Paperless-ng does use a bit more resources than Paperless. Running the web server requires about 300MB of RAM or more, depending on the configuration. While adding documents, it requires about 300MB additional RAM, depending on the document. It still runs on Raspberry Pi (many users do that), but it has been generally geared to better use the resources of more powerful systems.
-* API changes: If you rely on the REST API of paperless, some of its functionality has been changed.
-
-For a detailed list of changes done in paperless-ng, have a look at the [change log](https://paperless-ng.readthedocs.io/en/latest/changelog.html) in the documentation, especially the section about the [0.9.0 release](https://paperless-ng.readthedocs.io/en/latest/changelog.html#paperless-ng-0-9-0).
-
-Paperless-ngx forked from paperless-ng to continue the great work already done and distribute responsibility among a team of people.
-
-Discussion around that can be found in the issues in the paperless-ng repository:
-[1599](https://github.com/jonaswinkler/paperless-ng/issues/1599)
-[1632](https://github.com/jonaswinkler/paperless-ng/issues/1632)
-## Get in Touch
-
-People interested in continuing the work on paperless-ng(x) and form the organisation connected here on github and created a
-[Matrix Room](https://matrix.to/#/#paperless:adnidor.de) for realtime communication.
+![Paperless logo](https://github.com/paperless-ngx/paperless-ngx/raw/main/resources/logo/web/png/Black%20logo%20-%20no%20background.png#gh-light-mode-only)
```

Comment:
Does anyone know how we can control the size of the logo but still use the awesome `#gh-dark-mode-only` feature? 

I dont want to do anything too hack-y but most of what I read says have to use an `<img>` tag for sizing, which I think breaks `#gh-dark-mode-only`.